### PR TITLE
BXMSPROD-1265: Conscientious language initiative [master]

### PR DESCRIPTION
**Pull request by mbiarnes**
**Thank you for submitting this pull request**

**JIRA**: [BXMSPROD-1265](https://issues.redhat.com/browse/BXMSPROD-1265)

**referenced Pull Requests**:
https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1704
https://github.com/kiegroup/kie-jenkins-scripts/pull/1040
https://github.com/kiegroup/jenkins-pipeline-shared-libraries/pull/160
https://github.com/kiegroup/kie-docs/pull/3806
https://gitlab.cee.redhat.com/middleware/build-configurations/-/merge_requests/503

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest</b> (using <i>this</i> e.g. <b>Jenkins retest this</b> optional but no longer required)
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

**Review by mareknovotny — APPROVED** [review]

File: .ci/jenkins/Jenkinsfile.nightly [hunk 1]
```diff
@@ -39,7 +39,7 @@ pipeline {
                     def pipelineHelper = new PipelineHelper(this)
                     pipelineHelper.retry(
                     {
-                        def SETTINGS_XML_ID = "rhba-prod-master"
+                        def SETTINGS_XML_ID = "rhba-prod-main"
```
Comment (Ginxo) [inline .ci/jenkins/Jenkinsfile.nightly:42]: a pending change on https://gitlab.cee.redhat.com/business-automation/eng-jenkins/-/blob/master/secrets/managed-files_rhba-prod-master

File: pom.xml [hunk 2]
```diff
@@ -786,7 +786,7 @@
                       <ignoreClass>org.hornetq.utils.HornetQUtilBundle_$bundle</ignoreClass>
                       <ignoreClass>org.hornetq.utils.HornetQUtilLogger_$logger</ignoreClass>
                       <!-- Classes bundled in errai-javax-enterprise which are GWT friendly.
-                           See https://github.com/errai/errai/blob/master/errai-javax-enterprise/README -->
+                           See https://github.com/errai/errai/blob/main/errai-javax-enterprise/README -->
```
Comment (Ginxo) [inline pom.xml:789]: are we going to rename errai repository default branch? :thinking:

**Review by Ginxo — APPROVED** [review]
LGTM, just few comments to consider

File: pom.xml (hunk 2 above)
Comment (Ginxo) [reply pom.xml:789]: I see it will https://gist.github.com/mbiarnes/8987895c53e6ee352ba4ccdb78cab439
sorry @mbiarnes

Comment (mbiarnes) [reply pom.xml:789]: are we not? 
Will speak to Tiago.

Comment (Ginxo) [reply pom.xml:789]: @mbiarnes as I told you on the internal chat it is my fault. sorry

File: .ci/jenkins/Jenkinsfile.nightly (hunk 1 above)
Comment (mbiarnes) [reply .ci/jenkins/Jenkinsfile.nightly:42]: @Ginxo 
https://gitlab.cee.redhat.com/business-automation/eng-jenkins/-/merge_requests/80

**Pull request merged**